<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_read</class>
 <widget class="QDialog" name="Dialog_read">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Choose a button to show:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="read_a">
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_b">
         <property name="text">
          <string>Person</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_c">
         <property name="text">
          <string>Commmercial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_d">
         <property name="text">
          <string>Office</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_e">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
